test/test_helper/load.bash

```diff
@@ -166,11 +168,12 @@ generate_inputs() {
       end'
 		new_file="${file/$TEST_SUITE_INPUTS/$directory}"
 		mkdir -p "$(dirname "$new_file")"
-		sed_replace="s/<PROJECT>/$TEST_PROJECT/g"
+		sed_project_replace="s/<PROJECT>/$TEST_PROJECT/g"
+		sed_future_time_replace="s/<NEXT_DAY_TIME>/$NEXT_DAY_TIME/g"
```

Comment:
you can also separate sed commands in a single sed script using semi-colons (not sure it will work on mac):
```
echo '<PROJECT> some text <NEXT_DAY_TIME>' | sed 's/<PROJECT>/default/g ; s/<NEXT_DAY_TIME>/2024-02-01T12:00:00/g'
```